orchestrator/Dockerfile

```diff
@@ -29,14 +29,14 @@ RUN export GNUPGHOME="$(mktemp -d)" \
         && percona-release enable-only pdps-8.0 release
 
 RUN set -ex; \
+    microdnf update -y krb5-libs; \
     microdnf install -y \
         shadow-utils \
         percona-orchestrator \
         which \
         tar \
         openssl \
         procps-ng \
-        vim-minimal \
```

Comment:
let's not remove vim, like we did in router

Reply:
agree